### ISSUE TITLE
Entering settings on Edge

### ISSUE THREAD

**Issue by younyokel** [re621/re621#5]
Hello. I tried entering Settings using Edge (105.0.1343.27) and version 4.16.1 of Tampermonkey and got this error:

```
RE621 v.2.0.6 for Tampermonkey v.4.16.1
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36 Edg/105.0.1343.27
[SettingsManager] Fatal crash during "create"
TypeError: Cannot read properties of undefined (reading 'Settings')
    at SettingsManager.makeThumbnailSection (chrome-extension://iikmkjmpaadaobahmlepeloendndfphd/userscript.html?name=RE621%253A%2520e621%2520Enhancement%2520Suite.user.js&id=dc1eeab2-1b4f-4aeb-b783-b8fcdcf70212:63:134833)
    at SettingsManager.<anonymous> (chrome-extension://iikmkjmpaadaobahmlepeloendndfphd/userscript.html?name=RE621%253A%2520e621%2520Enhancement%2520Suite.user.js&id=dc1eeab2-1b4f-4aeb-b783-b8fcdcf70212:63:125640)
    at Generator.next (<anonymous>)
    at chrome-extension://iikmkjmpaadaobahmlepeloendndfphd/userscript.html?name=RE621%253A%2520e621%2520Enhancement%2520Suite.user.js&id=dc1eeab2-1b4f-4aeb-b783-b8fcdcf70212:63:125235
    at new Promise (<anonymous>)
    at G (chrome-extension://iikmkjmpaadaobahmlepeloendndfphd/userscript.html?name=RE621%253A%2520e621%2520Enhancement%2520Suite.user.js&id=dc1eeab2-1b4f-4aeb-b783-b8fcdcf70212:63:124980)
    at SettingsManager.create (chrome-extension://iikmkjmpaadaobahmlepeloendndfphd/userscript.html?name=RE621%253A%2520e621%2520Enhancement%2520Suite.user.js&id=dc1eeab2-1b4f-4aeb-b783-b8fcdcf70212:63:125383)
    at SettingsManager.<anonymous> (chrome-extension://iikmkjmpaadaobahmlepeloendndfphd/userscript.html?name=RE621%253A%2520e621%2520Enhancement%2520Suite.user.js&id=dc1eeab2-1b4f-4aeb-b783-b8fcdcf70212:63:51381)
    at Generator.next (<anonymous>)
    at chrome-extension://iikmkjmpaadaobahmlepeloendndfphd/userscript.html?name=RE621%253A%2520e621%2520Enhancement%2520Suite.user.js&id=dc1eeab2-1b4f-4aeb-b783-b8fcdcf70212:63:48819
```
What can be the problem?

**Comment by bitWolfy** [comment]
Oof. Yeah, I broke the script during the last patch by renaming one of the components.
Should be fixed in version 2.0.7.

**Comment by younyokel** [comment]
> Oof. Yeah, I broke the script during the last patch by renaming one of the components. Should be fixed in version 2.0.7.

Okay, thanks for clarifying!
